### ISSUE TITLE
Not Found after logging in

### ISSUE THREAD

**Issue by Haroenv** [bullgit/chitchat#35]
However clearing the url and going back to the base and doing it a second time always works.

**Comment by JessWallace94** [comment]
We need to find a way to ensure that it doesn't happen everytime, any reason as to why it's failing?

**Comment by LukyVj** [comment]
Yes, it's because of the https protocol, just make sure you use https while logging and then you shouldn't experience any problem :)
